freeswitch/freeswitch.xml

```diff
@@ -34,7 +34,6 @@
     <X-PRE-PROCESS cmd="set" data="recordings_dir=/tmp/"/>
     <X-PRE-PROCESS cmd="set" data="send_silence_when_idle=400"/>
     <X-PRE-PROCESS cmd="set" data="sdp_m_per_ptime=false"/>
-    <!--<X-PRE-PROCESS cmd="set" data="rtcp_mux=false"/>-->
```

Comment:
@kalda341 Hi, we don't want to enable rtcp_mux by default. we should be removing only the comment but leaving the value

Reply:
@BorigTheDwarf what's your objection ?

Reply:
I was more expressing general surprise at the necessity of this pull request rather than your response.

Reply:
@BorigTheDwarf `freeswitch` does a weird thing when it processes the `x-pre-process` elements. it does it by search/replace the returned body before it parses the xml. the result is that, even if you comment the `x-pre-process` elements they are still processed.

Reply:
@lazedo I've added it back in

Reply:
@kalda341 thanks for that. i'm evaluating this in different ways (tls/wss/tcp/udp/srtp/zrtp/rtp/rtcp) and in the end we may end up by actually removing it (or setting it explicit to `true` for visibility). the rtcp_mux for outbound that i tried & worked is actually part of a custom freeswitch build that fixes a bug where the rtcp_mux is set on the channel if the incoming invite has rtcp-mux in the sdp, but we may not release that build any time soon, so i'm evaluating the impact in other apps like fax between account (loopback involved) where this seemed to be an issue that was fixed.

thanks for your patience on this

Reply:
Hello @lazedo @kalda341 
I think rtcp_mux need to set false by default because not all equipment and software support this technology but support standard RTCP.

Please also look  [FS-10981](https://freeswitch.org/jira/browse/FS-10981). Here is described why WebRTC session fail if RTCP is disabled.